lsp-julia.el

```diff
@@ -62,7 +62,7 @@ Set to nil if you want to use the globally installed versions."
   :type 'string
   :group 'lsp-julia)
 
-(defcustom lsp-julia-default-environment "~/.julia/environments/v1.0"
+(defcustom lsp-julia-default-environment "~/.julia/environments/v1.2"
```

Comment:
I am not sure about this one, I think LanguageServer.jl aims to support Julia 1.0 and the latest stable release, so there are good reasons for both values. Ideally we would query the julia version in use, e.g. with
```
julia -e 'print("$(VERSION.major).$(VERSION.minor)")'
```
Not sure if it would be enough to query this only once at emacs startup, or we give `lsp-julia-default-environment` a `'default` value that queries this every time the language server is started.

Reply:
Yeah, that is much more elegant than simply hardcoding it! It would even be possible to do a compatibility check elsewhere if we are going that route.
I also like querying that value every time the server is started as it would allow switching versions more easily.
But I guess that warrants another PR. Shall I simply remove the change to `lsp-julia-default-environment` for now so you can merge this?

Reply:
ok, that sound good to me.

Reply:
Another thing to keep in mind is that the default depot being at `~/.julia` is specific to *nix afaik. While working on this, it may be worthwhile to add some more sophisticated detection there. Might be as simple as `julia -e print(DEPOT_PATH[1])`.

Reply:
```
julia -e 'print(joinpath(DEPOT_PATH[1], "$(VERSION.major).$(VERSION.minor)"))'
```